tests/main/preseed-lxd/task.yaml

```diff
@@ -51,7 +51,11 @@ prepare: |
 
   # for images that are already preseeded, we need to undo the preseeding there
   echo "Running preseed --reset for already preseeded cloud images"
-  SNAPD_DEBUG=1 /usr/lib/snapd/snap-preseed --reset "$IMAGE_MOUNTPOINT"
+  SNAP_PRESEED=/usr/lib/snapd/snap-preseed
+  if tests.info is-snapd-pkg-repo; then
+      SNAP_PRESEED="/snap/snapd/current$SNAP_PRESEED"
```

Comment:
Not a big deal since this test only runs on 18.04 but this should use snapd snap mount dir.

Reply:
done